Sources/BraintreeCore/BTClientMetadata.swift

```diff
@@ -30,18 +30,6 @@ public class BTClientMetadata: Encodable {
     
     public var version = BTCoreConstants.braintreeSDKVersion
     
-    // TODO: - Remove once all POSTs moved to Codable
-    /// Additional metadata parameters
-    var parameters: [String: Any] {
```

Comment:
This is tested in `testParameters_ReturnsTheMetadataMetaParametersForPosting` and still working as expected